kaiax/gov/impl/getter.go

```diff
@@ -26,3 +28,25 @@ func (m *GovModule) EffectiveParamSet(blockNum uint64) gov.ParamSet {
 
 	return *ret
 }
+
+func (m *GovModule) distill(gp gov.ParamSet, blockNum uint64) gov.ParamSet {
```

Comment:
can you make it a pure function and put it in `api.go`?
i.e. `distill(rule, gp) -> ParamSet`

Reply:
Done. https://github.com/kaiachain/kaia/pull/174/commits/406cfcf91ddf731d5f86fb0bfd15f5abda7e88d0